### ISSUE TITLE
Translate "Office of Police Oversight" in Spanish emails

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#1363]
**Describe the bug**
We translate "Office of Police Oversight" on alpha and on the forms—to be consistent, should translate in the form emails as well. Seems like low priority.

![unknown-5](https://user-images.githubusercontent.com/5839508/53509929-7d11f880-3a82-11e9-8449-3e35e3e687d2.jpeg)

**Comment by desigonz** [comment]
@easherma @amenity Looks like this is a "code, not copy" issue! "Office of Police Oversight" is baked into the template at a higher level than you're doing translations: https://github.com/cityofaustin/form-service-api/blob/7f7a7847fab69e12721146dbf92528f85f89feae/templates/email/officepoliceoversight/common/header.html#L7

**Comment by sergiogcx** [comment]
I got it, my bad. I have not been paying attention to that.

**Comment by sergiogcx** [comment]
I created a PR, 
https://github.com/cityofaustin/form-service-api/pull/13

Waiting on build to test and merge to master.

**Comment by sergiogcx** [comment]
![Screen Shot 2019-03-14 at 4.31.36 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/37f95a91-a6d2-4148-a028-09e599a83acf)

Fixed! :)

**Comment by sergiogcx** [comment]
Merged to master btw
